happypose/pose_estimators/cosypose/cosypose/datasets/bop.py

```diff
@@ -132,7 +132,7 @@ def __getitem__(self, frame_id):
                 RCO = np.array(annotation[n]["cam_R_m2c"]).reshape(3, 3)
                 tCO = np.array(annotation[n]["cam_t_m2c"]) * 0.001
                 TCO = Transform(RCO, tCO)
-                T0O = T0C * TCO
```

Comment:
I don't think this fix makes sense. 
`T0C` and `TCO` are at this point of type `Transform` defined [here](https://github.com/agimus-project/happypose/blob/dev/happypose/toolbox/lib3d/transform.py). This type implements the `__mul__` operator as multiplication `*` between the underlying `pin.SE3` objects, which corresponds to matrix multiplication betweeen the two homogeneous matrices. 
On the other hand, `__matmul__` is not implemented, which should result in a runtime error.